### ISSUE TITLE
Link-Profile-Picture construction error

### ISSUE THREAD

**Issue by Masono03** [Inve1951/BetterDiscordStuff#183]
Says: Missing start or stop function.

Canary 140672 (156c86e)
BetterDiscord 1.6.0
Host 1.0.48 Windows 10 64-Bit (10.0.19044)

**Comment by Keler69** [comment]
same problem after 1.6 update

**Comment by shamshitty** [comment]
yeah im getting this too after the 1.6 update

**Comment by aqtsalt** [comment]
same

**Comment by MGThePro** [comment]
adding an empty stop function at the end of the linkProfilePicture class seems to "fix" it.

something simple like this:
```
stop () {}
```

Probably not the ideal solution and I haven't tested this for very long but could be a temporary workaround

**Comment by Ketris** [comment]
I think the best solution is to copy _this.stop_ from line 15 into the added _stop(){}_, looking like:
```
  stop () {
	document.removeEventListener.bind(document, "click", LinkProfilePicture, true);
  }
```
This way the plugin can gracefully unload instead of doing nothing when asked to stop.

**Comment by BadScribbles** [comment]
> I think the best solution is to copy _this.stop_ from line 15 into the added _stop(){}_, looking like:
> 
> ```
>   stop () {
> 	document.removeEventListener.bind(document, "click", LinkProfilePicture, true);
>   }
> ```
> 
> This way the plugin can gracefully unload instead of doing nothing when asked to stop.

What a load of gibberish, I see why you made claims about my suggested Pull request being broken when it already worked for people.

You seem to have blindly copied and pasted code from the original plugin and when it didn't work, you blamed it on my code ? *which literally adds an empty function*

Your current suggested answer is the issue 

```
document.removeEventListener.bind(document, "click", LinkProfilePicture, true);
```
Maybe you didn't see the `bind()` call which returns a function but doesn't invoke it

Also, The stop function is overridden inside the start function. when the plugin stops is actually does clean up.

The suggested fix to only add en empty stop function to satisfy BD checks, the functionally is added later.

**Comment by Inve1951** [comment]
fixed in v1.2.4 (above PR)
